src/platform/utilities/feature-toggles/flipper-client.js

```diff
@@ -1,4 +1,4 @@
-const FEATURE_FLAG_NAMES = import('./featureFlagNames');
```

Comment:
dunno what i was thinking when i did this

Reply:
:joy: `require()` -> `import()`